app/app.go

```diff
@@ -370,7 +375,7 @@ func New(
 	bApp.SetInterfaceRegistry(interfaceRegistry)
 
 	keys := sdk.NewKVStoreKeys(
-		acltypes.StoreKey, authtypes.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey,
+		acltypes.StoreKey, authtypes.StoreKey, authzkeeper.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey,
```

Comment:
do we need to set beginblock/endblock ordering?
https://github.com/sei-protocol/sei-chain/blob/main/app/app.go#L631
https://github.com/sei-protocol/sei-chain/blob/main/app/app.go#L666

Reply:
We are doing so further down in this file